Simplenote/LoginRemote.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+// MARK: - LoginRemote
+//
+class LoginRemote: Remote {
```

Comment:
Is this that different from `AccountRemote`?  for the passkey remote things I dumped them in `AccountRemote` I don't mind the specifically targeted class, but 🤔

Reply:
(Belayed reply!!) This one is indeed different from AccountRemote. Account encapsulates Email Verification + Account Deletion.

I'd be super okay with merging them both, though. I'll revisit in a follow-up, TY!!